### ISSUE TITLE
(Feature Request): No punctuation at the end of text

### ISSUE THREAD

**Issue by Modifyinc** [futo-org/android-keyboard#97]
For texting, I love that it punctuates sentences automatically, but please, add an option to prevent punctuation from being added to the end of a text. It looks too formal. The question mark is ok, but periods at the end of a text is just too much.

**Comment by andranoxi** [comment]
The voice input is perfect for detecting commas mid-sentence, I love it for that. More often than not I'm deleting the period at the end of the transcriptions

**Comment by machinaeZER0** [comment]
This is specifically in regard to voice typing, correct? if so I agree - appreciate that it's picking up punctuation, and wouldn't want to lose question marks/commas, but i rarely end a text with a period. A toggle would be great, if possible!

**Comment by Modifyinc** [comment]
Yes, for voice typing. I know it's silly, but it is a pet peeve of mine when people end a text with a period. It just makes what usually is considered quick and laid back come across as something formal. For this reason, I gave up on this app for now. Hope to hear this will be addressed as I know I'm not the only one.

    On Tuesday, October 22, 2024 at 06:11:03 PM EDT, machinaeZER0 ***@***.***> wrote:  
 
 


This is specifically in regard to voice typing, correct? if so I agree - appreciate that it's picking up punctuation, and wouldn't want to lose question marks/commas, but i rarely end a text with a period. A toggle would be great, if possible!
